projects/plugins/jetpack/extensions/blocks/ai-paragraph/edit.js

```diff
@@ -204,8 +215,12 @@ export default function Edit( { attributes, setAttributes } ) {
 								setAttributes,
 								formattedPrompt,
 								setLoadingCompletion,
-								setErrorMessage
+								setErrorMessage,
+								postId
 							);
+							tracks.recordEvent( 'jetpack_ai_gpt3_completion_manual', {
+								post_id: postId,
+							} );
 						} }
```

Comment:
I have yet to play around with tracks. Can you clarify how I should be testing this? I see a query param for the `t.gif` request with the `_en`  and `jetpack_ai_gpt3_completion_automatic` value, but I'm unsure if this is the best way to test this functionality

Reply:
There are different ways to test this.

1. You could add the following to your browser console: `localStorage.setItem( 'debug', 'dops:analytics*' );`. You'll get to see all Tracks events there afterwards.
2. You can use the Tracks Vigilante browser extension. p7H4VZ-3cB-p2
3. You can search for the event in MC Tracks or search for all activity for your user; it can take 15-20 minutes to show, but it should show once you've clicked.